.travis.yml

```diff
@@ -7,5 +7,12 @@ node_js:
 matrix:
   allow_failures:
   - node_js: "10"
```

Comment:
that still true?

Reply:
I guess we no longer need this? though I don't know the reason why we added it in the first place

Reply:
maybe add this for node@12. circle stuff could be removed for now since we are using travis mainly

Reply:
@jcrugzz  do we delete the config.yml or the circleci webhook? or both